src/Microsoft.DotNet.Interactive.Formatting.Tests/FormatterTests.cs

```diff
@@ -214,27 +213,14 @@ public void Check_default_HTML_formatting_of_unicode()
                 text.Should().Be($"<div class=\"dni-plaintext\">{value.HtmlEncode()}</div>");
             }
 
-            [Fact]
-            public void When_JObject_is_formatted_it_outputs_its_string_representation()

```

Comment:
Is there an equivalent test we should add for System.Text.Json?

Reply:
we have those tests for the types in System.Text.Json